sympy/stats/tests/test_continuous_rv.py

```diff
@@ -1342,21 +1343,23 @@ def test_input_value_assertions():
 
 def test_unevaluated():
     X = Normal('x', 0, 1)
-    assert str(E(X, evaluate=False)) == ("Integral(sqrt(2)*x*exp(-x**2/2)/"
-    "(2*sqrt(pi)), (x, -oo, oo))")
+    with ignore_warnings(UserWarning):
+        assert str(E(X, evaluate=False).rewrite(Integral)) == ("Integral(sqrt(2)*x*exp(-x**2/2)/"
+        "(2*sqrt(pi)), (x, -oo, oo))")
 
-    assert str(E(X + 1, evaluate=False)) == ("Integral(sqrt(2)*x*exp(-x**2/2)/"
-    "(2*sqrt(pi)), (x, -oo, oo)) + 1")
+        assert str(E(X + 1, evaluate=False).rewrite(Integral)) == ("Integral(sqrt(2)*x*exp(-x**2/2)/"
```

Comment:
what about using `.dummy_eq( )` instead of comparing the strings? There was a bug with `.dummy_eq( )` some time ago, but it has been solved (I believe).